tests/VarDumperTest.php

```diff
@@ -218,15 +218,6 @@ static function () {
                 '🤣',
                 "'🤣'",
             ],
-            'closure in property supported' => [
```

Comment:
Why was it removed?

Reply:
> Why was it removed?

It isn't removed. It is moved. See below.